kubernetes/samples/scripts/create-oracle-db-service/README.md

```diff
@@ -0,0 +1,63 @@
+# Creating an Oracle Database service and RCU schema for a Fusion Middleware domain
```

Comment:
This sample is for creating Database service. RCU schema need to be removed from the heading.

Reply:
Modified the statement as follows  ...
Creating an Oracle Database Service for Fusion Middleware domain